.github/workflows/random-macosx-build.yml

```diff
@@ -69,6 +69,7 @@ jobs:
           ref: v${{ env.SOLIDITY_VERSION }}
           path: solidity/
           fetch-depth: 0
+          submodules: 'recursive'
```

Comment:
This action seems to be deactivated, but I added the option here anyway for sanity.